Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3.ps1

```diff
@@ -0,0 +1,1572 @@
+. $PSScriptRoot\CommonServerPowerShell.ps1
+<# IMPORTANT NOTICE
+# When conencting to ExchangeOnline - only needed command between CreateSession
+# and DisconnectSession and let also the `finally` term to disconnect (it will do nothing if everything is fine).
+# This will reduce the time sessions are opened between Exchange and the server and will create
+# less problems.
+# DO NOT USE ONE FINALLY STATEMENT: we don't know if and when it'll be executed and anyway it the DisconnectSession
```

Comment:
Does mean that the finally in main does not suffice ?
If so it means that all command calls need to be in try catch finally right ? 

If this is not the case the comment is not clear to me.

Reply:
I'm not entirely sure of the initial rationale. The last finally should take care of any connection issues unless the docker container is killed abruptly.